.github/workflows/onboarding_page_build.yml

```diff
@@ -0,0 +1,38 @@
+name: Onboarding Page Build
+
+on:
+  push:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.9"
+    - name: Install dependencies
+      run: |
+        sudo sed -i 's/yes/no/g' /etc/initramfs-tools/update-initramfs.conf
```

Comment:
It feels weird that you'd need to update the initramfs on machine that isn't going to be rebooted.

Reply:
Removing in 883f962ecea8c28fc00f8ffd217d6569a0b242f6